packages/dids/src/methods/did-method.ts

```diff
@@ -82,9 +82,9 @@ export interface DidCreateVerificationMethod<TKms> extends Pick<Partial<DidVerif
  * @typeparam O - The type of the options used for creating the DID.
  */
 export interface DidMethodApi<
-    TDid extends BearerDid,
-    TOptions extends DidCreateOptions<TKms>,
-    TKms extends CryptoApi | undefined = undefined
+    TKms extends CryptoApi | undefined = CryptoApi,
```

Comment:
any breaking changes in downstream packages expected?

Reply:
Nope -- this reduces the chances of any breaking changes by using a default that can be used (as opposed to `undefined`)